samples/csharp_dotnetcore/43.basic-bot-appinsights/basic-bot.bot

```diff
@@ -0,0 +1,46 @@
+﻿{

```

Comment:
Should this be checked in? Was the `basic-bot-appinsights.bot` file checked in before?